### ISSUE TITLE
testing the questions with moodle

### ISSUE THREAD

**Issue by djarecka** [ReproNim/module-dataprocessing#33]
@mjtravers - could you please check if our [xml file](https://github.com/ReproNim/module-dataprocessing/blob/gh-pages/quizzes/questions.xml) works fine with the moodle

**Comment by djarecka** [comment]
@mjtravers - have you had a chance to test the file I prepared?

**Comment by mjtravers** [comment]
@djarecka - I just uploaded it into moodle successfully. All 18 questions loaded.

Are these questions to be in a quiz at the end of the module or split between lessons?

**Comment by djarecka** [comment]
thanks @mjtravers ! Probably I should split this... Should I provide multiple xml files? What would be the easiest way for you?

**Comment by mjtravers** [comment]
@djarecka - Uploading in one big file is easiest. Can you provide a description of which questions should be in a quiz following each lesson? The actual quizzes in moodle will be created manually through the UI

**Comment by djarecka** [comment]
@mjtravers - below you have a list of questions for each lesson. I've noticed that one question was repeated. Also had to fix a bug, so please use the current xml: https://github.com/ReproNim/module-dataprocessing/blob/gh-pages/quizzes/questions.xml

lesson 1 (Core concepts using an analysis example):  qst. 1, 2
lesson 2 (Annotate, harmonize, clean, and version dat): qst. 3, 4, 5
Lesson 3 (Create and maintain reproducible computational environments): qst. 6, 7, 8, 9
Lesson 4( Create reusable and composable dataflow tools): qst. 10, 11, 12
Lesson 5 (Use integration testing to revalidate analyses as data and software change): qst. 13, 14
Lesson 6 (Track provenance from data to results): qst. 15, 16, 17

**Comment by djarecka** [comment]
@mjtravers - @satra and I trying to see the courses. It looks like we can't enroll and we have to be added. Could you please add us

**Comment by mjtravers** [comment]
@djarecka You are logged in and can't enroll?

**Comment by djarecka** [comment]
correct. @dbkeator suggested that this is still not open for public, so you have to add us

**Comment by mjtravers** [comment]
@djarecka - I have enrolled you and Satra in all 5 courses. Give it another try.

**Comment by djarecka** [comment]
still doesn't work

**Comment by mjtravers** [comment]
just sent you a Google Chat invite

**Comment by mjtravers** [comment]
@djarecka - The quizzes have been created and are ready for testing.
